_inc/client/components/jitm/index.jsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import {
+	isModuleActivated as _isModuleActivated,
+	activateModule,
+	isActivatingModule,
+} from 'state/modules';
+import Button from 'components/button';
+import { getJitm } from 'state/jitm';
+
+// to-do: bring in scss from jetpack-admin-jitm.scss. No need for now, but will be needed once we dequeue JITMs on the JP dashboard.
+
+class Jitm extends Component {
+	componentDidMount() {
+		analytics.tracks.recordEvent(
+			'jetpack_jitm_view',
+			// to-do: this does not work as we do not have the data when the component loads.
+			{ version: this.props.Jitm.id }
+		);
+	}
+
+	trackLearnMore = ( feature_class ) => {
+		analytics.tracks.recordEvent(
+			// to-do: set the right name and prop for that event.
+			'jetpack_jitm_view',
+			{ version: feature_class }
+		);
+	};
+
+	renderListItem = ( list, id, feature_class ) => {
+		for ( const i of list ) {
```

Comment:
let's use a more significant name here for `i`. even when it's straightforward, we should use `i` just on `for` loops, and/or standing for `index`.